docs/keycloak-configuration.md

```diff
@@ -26,153 +33,322 @@ To create a new realm follow the steps below:
 
 - Click on the "Master" realm drop-down on the top left side of the console as you can see below
 - Click on "Create Realm"
-- Give the realm a realm name - For example - "tornjak"
+
+![Keycloak Administration Console](/docs/rsrc/keycloak_diagrams/6-admin-console.png)
+
+- Give the realm a realm name - e.g. "Tornjak"
 - Make sure "Enabled" is set to On, as a realm that is not enabled can not be used
-- [Option 1]: The easiest way to create a realm is by using the import feature. To create the Tornjak realm download this (link for JSON file) file. And import to Keycloak by clicking on browse and import your resource file.
+
+![Create Realm Page](/docs/rsrc/keycloak_diagrams/7-create-realm-page.png)
+
+- Click on Create
+- Once the realm is created, you will see the realm name on the top left side of the console as you can see below. Here is where you can switch between different realms you created and visualize the Master Realm settings.
+
+![Tornjak Realm Page](/docs/rsrc/keycloak_diagrams/8-realm-page.png)
+
+# Option 1: Realm Import
+
+- The easiest way to create a realm is by using the import feature. To create the Tornjak realm download [this](examples/Tornjak-keycloak-realm-import.json) file. 
```

Comment:
Just a nit, this link should be (../examples/Tornjak-keycloak-realm-import.json)

Reply:
and instead of "this" put the actual file name